### PR TITLE
filtering out punctuation marks

### PR DESCRIPTION
**Pull request by rohankatyal29**

### CODE REVIEW

File: lib/scripts/mtw.js [hunk 1]
```diff
@@ -90,11 +90,13 @@ export class ContentScript {
 
   filter(countedWords, translationProbability, userDefinedTranslations, userBlacklistedWords) {
     var blackListReg = new RegExp(userBlacklistedWords);
+    var punctuationReg = new RegExp(/[.,\/#!$%\^&\*;:{}=\-_`~()]/g);
     var countedWordsList = this.shuffle(this.toList(countedWords, function(word, count) {
       return !!word && word.length >= 2 && // no words that are too short
         word !== '' && !/\d/.test(word) && // no empty words
         word.charAt(0) != word.charAt(0).toUpperCase() && // no proper nouns
         !blackListReg.test(word.toLowerCase()); // no blacklisted words
+        !punctuationReg.test(word.toLowerCase()); // no punctuation marks
```
Comment (Ceilican) [inline lib/scripts/mtw.js]: This is perfect for now. But could you add a comment in the code, saying that lines 98 and 99 could be optimized by merging the two regular expressions and testing only once?

**Comment by Ceilican** [comment]
Travis says that this PR is failing: https://travis-ci.org/OiWorld/MindTheWord/builds/133029346 .

Do you know why?

**Comment by Ceilican** [comment]
Well, anyway I merged it, and now Travis is behaving normally again: https://travis-ci.org/OiWorld/MindTheWord .

**Comment by Ceilican** [comment]
Thanks!!

**Comment by ankit-m** [comment]
@Ceilican Travis failed because @rohankatyal29 's branch did not have a travis.yml file. You can see the error 

yml

Rohan's branch did not have the commits I made. Hence Travis failed.

**Comment by rohankatyal29** [comment]
@ankit-m: I'll rebase with your repository once before the next PR, thanks!

**Pull request merged**